src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -74,6 +77,17 @@ public GpioController(PinNumberingScheme numberingScheme)
     /// </summary>
     public virtual int PinCount => _driver.PinCount;
 
+    /// <summary>
+    /// Returns the collection of open pins
+    /// </summary>
+    protected IEnumerable<GpioPin> OpenPins
```

Comment:
Can we make this internal instead? Or private protected? I don't want this API to be public or protected.